milhoja/core.py

```diff
@@ -82,9 +85,10 @@ def __exit__(self, type, value, traceback):
         # Remove worktree ref in upstream repository
         try:
             self.upstream.lookup_branch(self.name).delete()
-        except:
+        except Exception:
```

Comment:
Are we able to look for a specific exception here?

Reply:
Just going to remove the handling, not sure it's worthwhile silencing it.